src/game/setup.c

```diff
@@ -119,7 +119,7 @@ int32_t InitialiseLevel(int32_t level_num, GAMEFLOW_LEVEL_TYPE level_type)
 
     Overlay_Init();
 
-    g_HealthBarTimer = 100;
+    Overlay_SetHealtBarTimer(100); // TODO Move this to Overlay_Init()?
```

Comment:
Not sure. On one hand this is a good decision, on another here we're very explicit about flashing the health for 100 frames(?) at the start of the level. IMO let's leave it as-is, maybe move the blank line above under this line to lump `Overlay_` calls together.